app/subscriber/src/features/daily-overview/AVOverviewPreview.tsx

```diff
@@ -37,6 +58,80 @@ const AVOverviewPreview: React.FC = () => {
     handlePreviewReport();
   }, [handlePreviewReport]);
 
+  // build reference object for AV items that contain transcripts/videos
+  React.useEffect(() => {
+    if (instance && instance.sections?.length) {
+      instance.sections.forEach((section) => {
+        if (section && section.items?.length) {
+          section.items.forEach((item) => {
+            if (item?.contentId) {
+              const contentId = item?.contentId;
+              getContent(item?.contentId).then((content) => {
+                setItemMetadata({
+                  [contentId]: {
+                    hasTranscript:
+                      content?.contentType === ContentTypeName.AudioVideo && !!content?.body,
```

Comment:
The `content.isApproved=true` is required before it highlights a transcript is available.